### PR TITLE
Fix entities when geometry show interval is out of range

### LINKED ISSUE #2650: Undefined values for Entity boolean properties can crash Cesium
From [the forum](https://groups.google.com/d/msg/cesium-dev/wUL8pqy5jf0/0LQg4hRLTfIJ), the below snippet will crash Cesium when jumping to a time outside of the show interval because we internally return `undefined` which then has boolean logic applied to it, which simply propagated the undefined.  Eventually, we try and create a `ShowGeometryInstanceAttribute` with a value of undefined, which throws.

``` js
var czml = [{
        "id": "document",
        "version": "1.0"
}, {
    "id": "water1First",
    "availability": "2012-08-04T00:00:00Z/2012-08-05T02:00:54.9962195740191Z",
    "polygon": {
        "positions": {
            "cartographicDegrees": [
                0, 0, 55660,
                0.5, 0, 55660,
                0.5, 5, 55660,
                0, 5, 55660,
                0, 0, 55660
            ]
        },
        "material": {
            "solidColor": {
                "color": {
                    "rgba": [0, 0, 255, 255]
                }
            }
        },
        "outline": [
            {
                "boolean": true
            }
        ],
        "perPositionHeight": [
            {
                "boolean": true
            }
        ],
        "show": [
            {
                "interval": "2012-08-04T00:00:00Z/2012-08-04T02:00:54.9962195740191Z",
                "boolean": true
            }
        ]
    }
}];

var viewer = new Cesium.Viewer('cesiumContainer');
var dataSourcePromise = Cesium.CzmlDataSource.load(czml);
viewer.dataSources.add(dataSourcePromise);
viewer.zoomTo(dataSourcePromise);
```

### PR DESCRIPTION
**Pull request by hpinkos**
Fixes #2650

If the show interval is undefined for part of the availability, it now hides the geometry instead of crashing.

``` js
var czml = [{
        "id": "document",
        "version": "1.0"
}, {
    "id": "water1First",
    "availability": "2012-08-04T00:00:00Z/2012-08-05T02:00:54.9962195740191Z",
    "polygon": {
        "positions": {
            "cartographicDegrees": [
                0, 0, 55660,
                0.5, 0, 55660,
                0.5, 5, 55660,
                0, 5, 55660,
                0, 0, 55660
            ]
        },
        "material": {
            "solidColor": {
                "color": {
                    "rgba": [0, 0, 255, 255]
                }
            }
        },
        "outline": [
            {
                "boolean": true
            }
        ],
        "perPositionHeight": [
            {
                "boolean": true
            }
        ],
        "show": [
            {
                "interval": "2012-08-04T00:00:00Z/2012-08-04T02:00:54.9962195740191Z",
                "boolean": true
            }
        ]
    }
}];

var viewer = new Cesium.Viewer('cesiumContainer');
var dataSourcePromise = Cesium.CzmlDataSource.load(czml);
viewer.dataSources.add(dataSourcePromise);
viewer.zoomTo(dataSourcePromise);
```

### CODE REVIEW

File: Source/DataSources/GeometryUpdater.js [hunk 1]
```diff
@@ -317,7 +318,8 @@ define([
      */
     GeometryUpdater.prototype.isFilled = function(time) {
         var entity = this._entity;
-        return this._fillEnabled && entity.isAvailable(time) && this._showProperty.getValue(time) && this._fillProperty.getValue(time);
+        var visible = this._fillEnabled && entity.isAvailable(time) && this._showProperty.getValue(time) && this._fillProperty.getValue(time);
+        return Boolean(visible); // if show or fill are undefined, true && undefined === undefined
```
Comment (mramato) [inline Source/DataSources/GeometryUpdater.js]: Would it be more clear to use `defaultValue(visible, false)` thoughout?  Then you don't have to have the comment explaining what's going on.

Comment (hpinkos) [reply Source/DataSources/GeometryUpdater.js]: Sure, that works

**Comment by mramato** [comment]
Does it make more sense to default to false?  Since not defining show defaults to true does that make more sense?  I don't know the answer, just trying to reason my way to one and wondering what you were thinking?

**Comment by hpinkos** [comment]
My thinking is if show is undefined, the behavior is whatever we decide we want it to be, and undefined is usually false.  But I guess since we normally default show to true that could make sense too.  We chose to default color to white in this similar situation.

**Comment by mramato** [comment]
> My thinking is if show is undefined, the behavior is whatever we decide we want it to be, and undefined is usually false.

Sounds like clear enough reasoning to me.  I'll merge when green.  Thanks!

**Comment by hpinkos** [comment]
@mramato ready

**Pull request merged**